public/js/services/Web3Service.js

```diff
@@ -0,0 +1,20 @@
+var Web3Service = angular.module('Web3Service', [])
+    .service('web3', function () {
+        // Connect to Web3
+        try {
+            if (typeof web3 !== 'undefined') {
```

Comment:
We'll want this to point to our RPC endpoints, probably best to pass in via env var since we have 3 networks running.

Reply:
I don't think we can pass ENV vars to the front end in this app.